src/Microsoft.Diagnostics.Runtime/ClrModule.cs

```diff
@@ -253,7 +253,7 @@ private ulong GetSize()
         {
             ulong size = ExtendedData.Size;
             if (size != 0)
-                return 0;
+                return size;
```

Comment:
Wouldn't be cleaner to say `if (size == 0)` the whole block and return only once?